### DISCUSSION TITLE
Immediate Update Required

### DISCUSSION THREAD

**Discussion by HackrTP** [3kh0/ext-remover#562] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
After downgrading from the latest version and down to version 96, I accepted all the terms of services and it now says this

![ABB016C5-A308-4CCE-B96C-A8721221AAAF](https://user-images.githubusercontent.com/51166800/224187095-4a880103-7851-4152-92b4-d59dd44aa40c.jpeg)

**Comment by pl4udible** [comment]
idk man i think you need to redo it

**Comment by amelia-bartlett** [comment]


**Reply by Aka-but-nice** [reply, marked as answer]
No, in the OOBE, use "Control + Shift + E" To skip the updates, and sh1mmer was, by Google, announced patched but, it appears they've slipped up with the patch. In terms, theoretically, you can downgrade to continue using sh1mmer.
